From e926de6b1b4b7a703ab6c9bbfe50d5fc44a20390 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 12 Oct 2024 23:41:25 +0800
Subject: [PATCH 12/12] AOSCOS: fix(kmscon_conf.c): use stdout to print help
 information

Why, just why.

Using stderr would prevent `grep' or `less' from being used for output
filtering and paging, respectively.
---
 src/kmscon_conf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kmscon_conf.c b/src/kmscon_conf.c
index 7005155..a58001b 100644
--- a/src/kmscon_conf.c
+++ b/src/kmscon_conf.c
@@ -51,7 +51,7 @@ static void print_help()
 	 *       |10|    20   |    30   |    40   |    50   |    60   |    70   |    80   |
 	 *      "\t901234567890123456789012345678901234567890123456789012345678901234567890\n"
 	 */
-	fprintf(stderr,
+	fprintf(stdout,
 		"Usage:\n"
 		"\t%1$s [options]\n"
 		"\t%1$s -h [options]\n"
-- 
2.47.0

